### PR TITLE
Changes to documentation that came up while translating.

### PR DESCRIPTION
**Pull request by smoe**
I have translated a whopping 1% of the documentation to German and came across these issues.   @silopolis , I would love to get your comments, obviously, but most changes are not about style (except me prefering that table of G-codes) over something verbatim..

### CODE REVIEW

**Review by SebKuzminsky — CHANGES_REQUESTED** [review]
Thanks for these cleanups, I have two minor nitpicks.

**Comment by smoe** [comment]
There were are few more "manual mode"s that I also changed to "Manual mode".

File: docs/help/tklinuxcnc.adoc [hunk 1]
```diff
@@ -276,84 +276,86 @@ Manufacturing Sciences (NCMS) on the Next Generation Controller (NGC)
...
-G89                boring cycle, with dwell, feed out
-G90                non-incremental programming (the usual)
-G91                incremental programming
-G93                inverse time feed mode
-G94                feed per minute mode (the usual)
-G98,G99            set canned cycle return level
-----
+[cols="1,1"]
+|===
+|G00 X Y Z         | rapid feed |
```
Comment (hansu) [inline docs/help/tklinuxcnc.adoc:281]: Good idea to have this as table, but:
For all lines I think the last "|" is too much. At least according to the docs and my preview plug-in is also confused with this.
```suggestion
|G00 X Y Z         | rapid feed
```

And also for the following table.

**Review by hansu — CHANGES_REQUESTED** [review]

**Comment by silopolis** [comment]
Here they are.
Looks good to me 👌👍

File: docs/help/tklinuxcnc.adoc (hunk 1 above)
Comment (smoe) [reply docs/help/tklinuxcnc.adoc]: Fixed - and also installed a previewer.

**Comment by smoe** [comment]
You should not push --force when you have just accepted changes in the github web interface :o/
Thank you all!

**Comment by smoe** [comment]
@petterreinholdtsen - these will eventually trigger a first iteration on changes that need to be run against weblate. Besides not wanting to grow my diff too much, I really wanted to see this circle closed. I have made changes to a table that may be difficult. I have not changed the "FIXME"s in the documentation that were introduced for the syncing between languages when the same text appeared in different contexts.

**Comment by silopolis** [comment]
Le sam. 11 juin 2022 à 09:12, Hans ***@***.***> a écrit :

> Merged #1759 <https://github.com/LinuxCNC/linuxcnc/pull/1759> into master.
>

Drum roll!...

**Pull request merged**